checker/src/org/checkerframework/checker/index/IndexChecker.java

```diff
@@ -13,7 +13,9 @@
  * <p>Constant Value Checker, SameLen Checker, SearchIndex Checker, Lower Bound Checker, Upper Bound
  * Checker
  *
- * <p>The Constant Value Checker has no dependencies.
+ * <p>The Constant Value Checker has no dependencies, but it does trust Positive annotations from
```

Comment:
Mentioning it in ValueChecker.java seems reasonable, but I think it's important to mention it here, too, since this is documenting the way the Index Checker's subcheckers interact, and the Lower Bound - Value Checker interaction is very important.